extensions/amp-ad-network-doubleclick-impl/refresh.md

```diff
@@ -0,0 +1,99 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AMP Ad Refresh
```

Comment:
Can we break this into two documents?  One on how ad networks can implement refresh (ideally this would live under amp-a4a directory) and one for how pubs can use with amp-ad type=doubleclick.  Let's be clearer about the fact that refresh interval does not start until the slot is viewable and the interaction with SRA.

Reply:
Done, though I'm not sure what you mean by the interaction with SRA. Currently, we don't support both features on the same slot. This is documented at the bottom of the page.